.gitignore

```diff
@@ -1,40 +1 @@
-COPYING
-INSTALL
-README
-AUTHORS
-NEWS
-Makefile.in
-Makefile
-aclocal.m4
-autom4te.cache
-configure
-config.cache
-config.h
-config.log
-config.rpath
-config.status
-compile
-depcomp
-description-pak
-install-sh
-missing
-po/stamp-it
-src/gamesxml2c
-.*.swp
```

Comment:
`.*.swp` files are not build files, they are temporary files created by [vim, the editor](http://www.vim.org/). gitignoring them prevents users to commit temporary files while they edit some files.